doc/source/notebooks/understanding/tf_graphs_and_sessions.ipynb

```diff
@@ -25,14 +25,14 @@
    "metadata": {},
    "source": [
     "\n",
-    "TensorFlow gives you a default graph, which you fill with tensors and operations - nodes and edges in the graph respectively. You can find details [here](https://www.tensorflow.org/guide/graphs#building_a_tfgraph) on how to change the default graph to another one or exploit multiple graphs.\n",
+    "TensorFlow gives you a default graph, which you fill with tensors and operations - nodes and edges in the graph respectively. See the [TensorFlow documentation on building a `tf.Graph`](https://www.tensorflow.org/guide/graphs#building_a_tfgraph) for details on how to change the default graph or exploit multiple graphs.\n",
     "\n",
-    "TensorFlow graph is a representation of your computation, to execute your code you need a TensorFlow [session](https://www.tensorflow.org/guide/graphs#executing_a_graph_in_a_tfsession). For example, you can think of graphs and sessions as binary sources and actual command running it in  a terminal. Normally, TensorFlow doesn't provide the default session, although GPflow creates default session and you can get it via:"
+    "A TensorFlow graph is a representation of your computation. To execute your code you need a [TensorFlow session](https://www.tensorflow.org/guide/graphs#executing_a_graph_in_a_tfsession). For example, you can think of graphs and sessions as binary sources and actual command running it in  a terminal. Normally, TensorFlow doesn't provide the default session; however, GPflow creates a default session that you can access as shown in the following example:"
```

Comment:
"For example, you can think of graphs and sessions as binary sources and actual command running it in  a terminal."

This sentence doesn't seem clear to me - can someone clarify?